index.html

```diff
@@ -414,359 +342,358 @@ <h2>
-However, all implementations of a given <a>DID method</a> must be interoperable
-for that method.
-    </p>
-  </section>
-
-  </section>
-
-  <section class="informative">
-    <h1>
-Terminology
-    </h1>
-
-    <div data-include="terms.html" data-oninclude="restrictReferences">
-    </div>
-
-  </section>
-
-  <section>
-    <h1>Overall Architecture</h1>
-
-    <section>
-      <h2>DIDs</h2>
-    </section>
-
-    <section>
-      <h2>DID Registries</h2>
+            </td>
+          </tr>
+        </tbody>
+      </table>
     </section>
 
     <section>
-      <h2>DID Documents</h2>
+      <h2>
+Architecture Overview
+      </h2>
 
       <p>
-A <a>DID document</a> is the <a>resource</a> that is associated with a
-<a>decentralized identifier</a> (DID). <a>DID documents</a> typically express
-verification methods (such as public keys) and services that can be used to
-interact with a <a>DID controller</a>.
+This section provides a basic understanding of the major elements of DID
+architecture. Formal definitions of terms are provided in <a href="#terminology"></a>.
       </p>
 
-      <p>
-A <a>DID document</a> is serialized according to a particular syntax, as
-outlined in Section <a href="#core-representations"></a>). The <a>DID</a> itself
-is contained in the <code>id</code> property.
-      </p>
+      <dl>
+        <dt>
+DIDs and DID URLs
+        </dt>
+        <dd>
+A <a>DID</a>, or Decentralized Identifier, is a URI composed of three parts:
+the scheme "did:", a method identifier, and a unique identifier generated by
+the <a>DID method</a>. DIDs are resolvable to <a>DID documents</a>. A
+<a>DID URL</a> extends the syntax of a basic DID to incorporate other standard
+URI components (path, query, fragment) in order to locate a particular
+resource&mdash;for example, a public key inside a <a>DID document</a>, or a resource
+available external to the <a>DID document</a>.
+        </dd>
 
-      <p>
-The properties that can be present in a <a>DID document</a> are outlined in
-Section <a href="#did-documents"></a>.
-      </p>
+        <dt>
+DID Subjects
+        </dt>
+        <dd>
+The subject of a <a>DID</a> is, by definition, the entity identified by the
+<a>DID</a>. The <a>DID subject</a> may also be the <a>DID controller</a>.
+Anything can be the subject of a DID: person, group, organization, physical
+thing, logical thing, etc.
+        </dd>
 
-      <p>
-The properties present in a <a>DID document</a> can be updated according to
-the applicable operations outlined in Section <a href="#methods"></a>.
-      </p>
+        <dt>
+DID Controllers
+        </dt>
+        <dd>
+The <a>controller</a> of a <a>DID</a> is the entity (person or organization)
+that has the capability&mdash;as defined by a <a>DID method</a>&mdash;to make
+changes to a <a>DID document</a>. This capability is typically asserted by the
+control of a set of cryptographic keys, though it may also be asserted via
```

Comment:
```suggestion
control of a set of cryptographic keys used by software acting on behalf of the controller, though it may also be asserted via
```

Adding this here would address my concerns in #210

Reply:
I'm fine w/ the change as it makes what is using the cryptographic material more precise.